pkg/xmtp/message.go

```diff
@@ -27,40 +28,46 @@ func parseGroupMessage(groupMessage []byte) (*mlsV1.GroupMessage_V1, error) {
 	return v1Message, nil
 }
 
-func parseConversationMessage(message []byte) (*messageContents.MessageV2, error) {
-	var msg messageContents.Message
-	err := proto.Unmarshal(message, &msg)
-	if err != nil {
-		return nil, err
-	}
-
-	v2Message := msg.GetV2()
-	if v2Message != nil {
-		return v2Message, nil
-	}
-	return nil, errors.New("Not a V1 message")
-}
-
-func getContext(env *messageApi.Envelope) interfaces.MessageContext {
+func getContext(env *messageApi.Envelope, logger *zap.Logger) interfaces.MessageContext {
 	messageType := topics.GetMessageType(env)
 	var shouldPush *bool
 	var hmacInputs, senderHmac *[]byte
-	if messageType == topics.V2Conversation {
-		if parsed, err := parseConversationMessage(env.Message); err == nil {
-			shouldPush = parsed.ShouldPush
-			hmacInputs = &parsed.HeaderBytes
-			if len(parsed.SenderHmac) > 0 {
-				senderHmac = &parsed.SenderHmac
-			}
-		}
-	} else if messageType == topics.V3Conversation {
+
+	if messageType == topics.V3Conversation {
 		if message, err := parseGroupMessage(env.Message); err == nil {
-			push := true
-			shouldPush = &push
+			logger.Info("MESSAGE PARSED LOPI",
```

Comment:
We'll want to remove this before merging. Also I wouldn't think it'll compile until we either remove the logger from the arguments to this function, or pass the logger from all the callsites.